petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -478,3 +494,36 @@ def delay_create_file2():
     threading.Thread(target=delay_create_file2()).start()
 
     _wait_file_available(url_list)
+
+
+def test_check_dataset_file_median_size(test_ctx):
+    file_size_map = {
+        '/a/b/01.parquet': 50,
+        '/a/b/02.parquet': 70,
+        '/a/b/03.parquet': 60,
+        '/a/b/04.parquet': 65,
+        '/a/b/05.parquet': 999000,
+    }
+    with mock.patch('os.path.getsize') as mock_path_get_size:
+        mock_path_get_size.side_effect = lambda p: file_size_map[p]
+        url_list = ['file://' + path for path in file_size_map.keys()]
+        with patch_logger('petastorm.spark.spark_dataset_converter') as output:
```

Comment:
pytest appears to have a [built in log capture](https://docs.pytest.org/en/latest/logging.html). Do you think we should use it instead of a custom one?

Reply:
Oh sure if the builtin one works fine.